src/main/java/net/fabricmc/loader/gui/FabricStatusTree.java

```diff
@@ -203,10 +203,43 @@ public void setInfo() {
 			setWarningLevel(FabricTreeWarningLevel.INFO);
 		}
 
+		private FabricStatusNode addChild(String string, boolean info) {
+			if (string.startsWith("\t")) {
+				if (children.size() == 0) {
+					FabricStatusNode rootChild = new FabricStatusNode(this, "(indented node was added before root node - this is a bug!)");
+					rootChild.warningLevel = FabricTreeWarningLevel.WARN;
+					children.add(rootChild);
+				}
+				FabricStatusNode lastChild = children.get(children.size() - 1);
+				lastChild.addChild(string.substring(1), true);
+				return lastChild;
+			} else {
+				FabricStatusNode child = new FabricStatusNode(this, info ? emboldenForNode(string) : cleanForNode(string));
+				if (info) {
+					child.warningLevel = FabricTreeWarningLevel.INFO;
+					expandByDefault = true;
+				}
+				children.add(child);
+				return child;
+			}
+		}
+
 		public FabricStatusNode addChild(String string) {
-			FabricStatusNode child = new FabricStatusNode(this, string);
-			children.add(child);
-			return child;
+			return addChild(string, false);
+		}
+
+		private String cleanForNode(String string) {
+			string = string.trim();
+			if (string.length() > 1) {
+				if (string.startsWith("-"))
+					string = string.substring(1);
+				string = string.trim();
+			}
+			return string;
+		}
+
+		private String emboldenForNode(String string) {
+			return "<html><b>" + cleanForNode(string) + "</b></html>";
```

Comment:
This makes me suspicious of whether the code deals with string containing characters that are meaningful html.